### DISCUSSION TITLE
Need a new version 100% WHQL compliant

### DISCUSSION THREAD

**Discussion by Mushan3420** [pal1000/Realtek-UAD-generic#109] [General]
I am using 6.0.8856.1, which is 100% WHQL compliant, and I need a new version like that.

**Comment by pal1000** [comment]
Just remove `Realtek-UAD-generic\Win64\Realtek\UpdatedCodec`. This will disable force updater making setup install WHQL compliant components only.

**Reply by Mushan3420** [reply]
I prefer the official Realtek UAD installer, which automatically uninstalls the old driver and installs the new one.
